packages/app/src/core/plugin-assembler.ts

```diff
@@ -18,19 +18,17 @@ import * as Debug from 'debug'
 
 import * as fs from 'fs-extra'
 import * as path from 'path'
-import * as events from 'events'
-import { exec } from 'child_process'
 
 import * as plugins from './plugins'
 import * as commandTree from './command-tree'
 const debug = Debug('core/plugin-assembler')
-debug('loading')
-import mkdirp = require('mkdirp')
+// debug('loading')
 
-const TMP = 'plugins' // we'll stash the original plugins here
-const TMA = 'app' // we'll stash the original app here
+// // TODO
+// const TMP = 'plugins' // we'll stash the original plugins here
+// const TMA = 'app' // we'll stash the original app here
```

Comment:
Thanks! TMP and TMA are outdated indeed. It's ok to delete them! 😄

Reply:
The result after dropping those lines was a bit strange:

![image](https://user-images.githubusercontent.com/2753855/59417291-40ff6b00-8dc7-11e9-95f5-fe47f3c2ae7a.png)

So I changed it to this. Is what we want?

![image](https://user-images.githubusercontent.com/2753855/59417257-2dec9b00-8dc7-11e9-9280-a6c819d69bae.png)

Reply:
yes seems good to me, then we can know the module loading time. thanks!